src/differentiation/compute_jacobian_ad.jl

```diff
@@ -98,6 +98,7 @@ function forwarddiff_color_jacobian!(J::AbstractMatrix{<:Number},
     sparsity = jac_cache.sparsity
     color_i = 1
     chunksize = length(first(first(jac_cache.p)))
+    fill!(J,zero(eltype(J)))
```

Comment:
```suggestion
    fill!(J, zero(eltype(J)))
```

Reply:
Could also be a bit shorter with the eltype in the signature, with a `where` clause